test/extended/builds/multistage.go

```diff
@@ -24,6 +24,8 @@ FROM scratch as test
 USER 1001
 FROM centos:7
 COPY --from=test /usr/bin/curl /test/
+COPY --from=busybox:latest /bin/echo /test/
+COPY --from=busybox:latest /bin/ping /test/
```

Comment:
How will this be verified?

Reply:
Since bug 1684427, the content  "FROM busybox:latest"  in build-logs show serval times.After bug fix, I will check the step order, Do not like below:
STEP 1: FROM busybox:latest
STEP 2: FROM busybox:latest

Reply:
https://github.com/openshift/origin/pull/22589/files#diff-e21c55070c6c407d1d1f7938cf49fc63R113 
If this could be enough?